infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
```

Comment:
If the `SNS_ARN` environment variable is not set, this code will not complain.
The default value for `os.environ.get("SNS_ARN")` if the named item is not found is `None`. This may cause unexpected behaviour - publishing messages to a topic named `None:dev_gh_team_events`, for example.
Use dict access (`os.environ['SNS_ARN']`) instead so that the error will be raised at this point and will be self-explanatory.